crates/starknet-types-core/README.md

```diff
@@ -1,16 +1,33 @@
 # starknet-types-core
 
-`starknet-types-core` is a crate focusing on Starknet types related to computation and execution. This crate is part of an initiative to standardize the representation of the `Felt` type in Rust, reducing code complexity and improving performance across the Starknet Rust ecosystem.
+Core types representation for Starknet.
 
-The types in this crate require performance and optimization for specific arithmetic and cryptographic operations, making it ideal for computational tasks within the Starknet ecosystem.
+## Overview
+
+The `starknet-types-core` crate provides:
+* The universal `Felt` (Field Element) type for Cairo and STARK proofs. It was created to reduce the fragmentation in the Starknet Rust ecosystem by providing a standardized representation of the `Felt` type.
+* The `AffinePoint` and `ProjectivePoint` structs, that represent a point in the Stark curve with which you can operate
+
+## Features
+
+- Standardized `Felt` type: Simplify your codebase by using our standardized `Felt` type.
+- Optimized for performance: The `Felt` type has been optimized for high-performance applications.
+
+## Examples
+
+Here are some examples of how to use the `Felt` type:
+
+```rust
+    let x = Felt::from(18)
+```
```

Comment:
Should't there be at least a couple more simple examples?

Reply:
Done